docs/api/web-contents.md

```diff
@@ -588,6 +588,15 @@ Emitted when media starts playing.
 
 Emitted when media is paused or done playing.
 
+#### Event: 'audio-state-changed'
+
+Returns:
```

Comment:
Should this use the new event style https://github.com/electron/electron/pull/37085 ?

Reply:
good question! I don't feel strongly either way - i did this because the other PR is not yet merged. Happy to either

1) wait until it's merged and rebase/update accordingly
2) update it now

Which would you prefer?

Reply:
Question for @nornagon, once https://github.com/electron/electron/pull/37085 gets merged will there be a refactoring of existing events to make them consistent ?

My preference would be for option `1) wait until it's merged and rebase/update accordingly` if there is a planned refactor on the way.

Reply:
I don't know why that's a question for me, I posted that in our #wg-api channel as a question for the WG :)

Reply:
Sorry, I was trying to defer to your decision on whether this PR should use the new style already since I didn't see a final consensus on the wg-api discussion thread.

Reply:
done